src/patternfly/utilities/Text/text.scss

```
@@ -57,113 +48,101 @@ $pf-v6-u-font-size-options: (
 // Font weight options
 $pf-v6-u-font-weight-options: (
   font-weight-normal: (
-    font-weight var(--#{$pf-global}--FontWeight--normal)
+    font-weight var(--pf-t--global--font--weight--body)
   ),
   font-weight-bold: (
-    font-weight var(--#{$pf-global}--FontWeight--bold)
+    font-weight var(--pf-t--global--font--weight--body--bold)
   )
 );
 
 @include pf-v6-utility-builder($pf-v6-u-font-weight-options, $pf-v6-global--breakpoint-list);
 
 // Font color options
 $pf-v6-u-font-color-options: (
-  color-100: (
-    color var(--#{$pf-global}--Color--100)
-  ),
-  color-200: (
-    color var(--#{$pf-global}--Color--200)
+  text-color-brand-default: (
```

Just to be consistent, I'd change this to

```suggestion
  text-color-brand: (
```